### PR TITLE
Add ArtifexSoftware.mutool

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107568)

### CODE REVIEW

**Comment by kbdharun** [comment]
I am sure the installer URL/domain is correct and I have tested the PR's manifest locally and it works as expected.

Maybe this might be the case.

> This may happen for sites with policies prohibiting distribution or use by third parties.

>> Regarding the validation executable error, I will try adding uninstall behaviour.

**Comment by stephengillie** [comment]
This package is expecting arguments:
```
mutool version 1.22.1
usage: mutool <command> [options]
        clean   -- rewrite pdf file
        convert -- convert document
        create  -- create pdf document
        draw    -- convert document
        trace   -- trace device calls
        extract -- extract font and image resources
        info    -- show information about pdf resources
        merge   -- merge pages from multiple pdf sources into a new pdf
        pages   -- show information about pdf pages
...
```

**Review by stephengillie — DISMISSED** [review]

File: manifests/a/ArtifexSoftware/mutool/1.22.1/ArtifexSoftware.mutool.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.2.6 $debug=NVS1.CRLF.5-1-22621-1037.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ArtifexSoftware.mutool
+PackageVersion: 1.22.1
+PackageLocale: en-US
+Publisher: Artifex Software
+PublisherUrl: https://www.mupdf.com
```
Comment (stephengillie) [inline manifests/a/ArtifexSoftware/mutool/1.22.1/ArtifexSoftware.mutool.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://artifex.com/
```

File: manifests/a/ArtifexSoftware/mutool/1.22.1/ArtifexSoftware.mutool.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+PublisherSupportUrl: https://artifex.com/support
+PrivacyUrl: https://www.mupdf.com/privacy/index.html
+Author: Artifex Software
+PackageName: mutool
+PackageUrl: https://github.com/ArtifexSoftware/mupdf
```
Comment (stephengillie) [inline manifests/a/ArtifexSoftware/mutool/1.22.1/ArtifexSoftware.mutool.locale.en-US.yaml:13]:
```suggestion
PackageUrl: https://www.mupdf.com/
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**